huaweicloud/config/hc_config.go

```diff
@@ -134,92 +135,49 @@ func getServiceEndpoint(c *Config, srv, region string) string {
 }
 
 // NewVpcClient is the VPC service client using huaweicloud-sdk-go-v3 package
-func NewVpcClient(c *Config, region string) (*vpc.VpcClient, error) {
-	credentials, err := buildAuthCredentials(c, region)
+func (c *Config) HcVpcV3Client(region string) (*vpcv3.VpcClient, error) {
```

Comment:
how about keeping the origin func name and pattern, then we don't need to modify the reference place.

Reply:
version is necessary and maybe it is easier to identify with the prefix `Hc`